numpyro/infer/renyi_elbo.py

```diff
@@ -0,0 +1,71 @@
+from jax import random, vmap
+from jax.lax import stop_gradient
+from jax.scipy.linalg import logsumexp
+import jax.numpy as np
+from numpyro.infer.elbo import ELBO
+from numpyro.infer.util import log_density
+from numpyro.handlers import replay, seed
+
+
+class RenyiELBO(ELBO):
+    r"""
+    An implementation of Renyi's :math:`\alpha`-divergence variational inference
+    following reference [1].
+    In order for the objective to be a strict lower bound, we require
+    :math:`\alpha \ge 0`. Note, however, that according to reference [1], depending
+    on the dataset :math:`\alpha < 0` might give better results. In the special case
+    :math:`\alpha = 0`, the objective function is that of the important weighted
+    autoencoder derived in reference [2].
+    .. note:: Setting :math:`\alpha < 1` gives a better bound than the usual ELBO.
+        For :math:`\alpha = 1`, it is better to use
+        :class:`~pyro.infer.trace_elbo.Trace_ELBO` class because it helps reduce
+        variances of gradient estimations.
+    .. warning:: Mini-batch training is not supported yet.
```

Comment:
to make sure that this is rendered correctly, you can use `make docs` and checkout `docs/build` folder (after you have added RenyiELBO to docs)